x-pack/plugins/watcher/public/components/delete_watches_modal.tsx

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { EuiConfirmModal, EuiOverlayMask } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+import { toastNotifications } from 'ui/notify';
+import { deleteWatches } from '../lib/api';
+
+export const DeleteWatchesModal = ({
+  watchesToDelete,
+  callback,
+}: {
+  watchesToDelete: string[];
+  callback: (deleted?: string[]) => void;
+}) => {
+  const numWatchesToDelete = watchesToDelete.length;
+  if (!numWatchesToDelete) {
+    return null;
+  }
+  const confirmModalText = i18n.translate(
+    'xpack.watcher.deleteSelectedWatchesConfirmModal.descriptionText',
+    {
+      defaultMessage:
+        'This will permanently delete {numWatchesToDelete, plural, one {a watch} other {# watches}}. Are you sure?',
+      values: { numWatchesToDelete },
+    }
+  );
+  const confirmButtonText = i18n.translate(
+    'xpack.watcher.deleteSelectedWatchesConfirmModal.deleteButtonLabel',
+    {
+      defaultMessage: 'Delete {numWatchesToDelete, plural, one {watch} other {# watches}} ',
+      values: { numWatchesToDelete },
+    }
+  );
+  const cancelButtonText = i18n.translate(
+    'xpack.watcher.deleteSelectedWatchesConfirmModal.cancelButtonLabel',
+    {
+      defaultMessage: 'Cancel',
+    }
+  );
+  return (
+    <EuiOverlayMask>
+      <EuiConfirmModal
+        title={confirmButtonText}
+        onCancel={() => callback()}
+        onConfirm={async () => {
+          const { successes, errors } = await deleteWatches(watchesToDelete);
+          const numSuccesses = successes.length;
+          const numErrors = errors.length;
+          callback(successes);
+          if (numSuccesses > 0) {
+            toastNotifications.addSuccess(
+              i18n.translate(
+                'xpack.watcher.sections.watchList.deleteSelectedWatchesSuccessNotification.descriptionText',
+                {
+                  defaultMessage:
+                    'Deleted{numWatchesToDelete, plural, one {watch} other {{numSuccesses} out of # selected watches}} ',
```

Comment:
I think we need to add a space after "Deleted":

```js
'Deleted {numWatchesToDelete, plural, one {watch} other {{numSuccesses} out of # selected watches}} ',
```

Reply:
fixed